src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/assessRisksAndNeeds/NeedsTest.kt

```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.assessRisksAndNeeds
+
+import io.kotest.core.spec.style.DescribeSpec
+import io.kotest.matchers.shouldBe
+import java.time.LocalDateTime
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.assessRisksAndNeeds.Needs as ArnNeeds
+
+class NeedsTest : DescribeSpec(
+  {
+    describe("#toNeeds") {
+      it("maps ARN Needs to integration API Needs") {
```

Comment:
Usually for 1:1s we use "maps one-to-one attributes to Integration API attributes"